src/components/ButtonPanel.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = () => (
+  <div>
+    <div>
+      <Button name="AC" />
+      <Button name="+/-" />
+      <Button name="%" />
+      <Button name="÷" />
+    </div>
+    <div>
+      <Button name="7" />
+      <Button name="8" />
+      <Button name="9" />
+      <Button name="X" />
+    </div>
+    <div>
+      <Button name="4" />
+      <Button name="5" />
+      <Button name="6" />
+      <Button name="-" />
+    </div>
+    <div>
+      <Button name="1" />
+      <Button name="2" />
+      <Button name="3" />
+      <Button name="+" />
+    </div>
+    <div>
+      <Button name="0" />
+      <Button name="." />
+      <Button name="=" />
```

Comment:
- [ ] OPTIONAL - Rather than doing this. Simple put the button name in an array of objects and map through them. To avoid repetition.
```
let buttonObj = [{
  name: '0',
}]
```